ipfs-api-prelude/src/api.rs

```diff
@@ -2004,6 +2005,138 @@ pub trait IpfsApi: Backend {
 
     // TODO /pin/verify
 
+    /// Pin object to remote pinning service.
+    ///
+    /// - `service`: Name of the remote pinning service to use (mandatory).
+    ///
+    /// - `name`: An optional name for the pin.
+    ///
+    /// - `background`: Add to the queue on the remote service and
+    /// return immediately (does not wait for pinned status).
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use ipfs_api::{IpfsApi, IpfsClient};
+    ///
+    /// let client = IpfsClient::default();
+    /// let res = client.pin_remote_add("QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ", "pinata", Some("pin_task_name"), true);
+    /// ```
+    ///
```

Comment:
trim this line.

Reply:
Sorry I didn't understand what you meant, did you want me to delete the blank line?

Reply:
> did you want me to delete the blank line?

Yep

Reply:
Looks like every api doc has a blank line in the end

Reply:
Let's keep the same style if so.

Leave this decision for @ferristseng to make.

Reply:
Yeah, I think this is fine. I am really bad about being consistent with this, but if it matches other places then I think it makes sense to leave it.